scaling_performance/optimizing_storage.adoc

```diff
@@ -21,11 +21,11 @@ limitations of a union file system, and does not yet support SELinux.
 
 For more information about Overlay, including supportability and usage caveats,
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/7.3_Release_Notes/index.html#technology_previews_file_systems[see
-the RHEL 7.3 Release Notes].
+the Red Hat Enterprise Linux (RHEL) 7 Release Notes].
 
 In production environments, using a LVM thin pool on top of regular block
 devices (not loop devices) for container images and container root file systems
-storage is recommended. 
+storage is recommended.
```

Comment:
Is "root file systems storage" correct? Just sounds odd.
"root file system storage"?